docs/core/testing/unit-testing-visual-basic-with-nunit.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Unit testing Visual Basic in .NET Core using dotnet test and NUnit
+description: Learn unit test concepts in .NET Core through an interactive experience building a sample Visual Basic solution step-by-step using NUnit.
+author: rprouse
+ms.date: 12/01/2017
+ms.topic: article
+dev_langs:
+ - "vb"
+ms.prod: .net-core
+---
+# Unit testing Visual Basic .NET Core libraries using dotnet test and NUnit
+
+This tutorial takes you through an interactive experience building a sample solution step-by-step to learn unit testing concepts. If you prefer to follow the tutorial using a pre-built solution, [view or download the sample code](https://github.com/dotnet/docs/tree/master/samples/core/getting-started/unit-testing-vb-nunit/) before you begin. For download instructions, see [Samples and Tutorials](../../samples-and-tutorials/index.md#viewing-and-downloading-samples).
+
+## Creating the source project
+
+Open a shell window. Create a directory called *unit-testing-vb-nunit* to hold the solution. Inside this new directory, run [`dotnet new sln`](../tools/dotnet-new.md) to create a new solution. This practice
+makes it easier to manage both the class library and the unit test project. Inside the solution directory, create a *PrimeService* directory. You have the following directory and file structure thus far:
+
+```
+/unit-testing-vb-nunit
+    unit-testing-vb-nunit.sln
+    /PrimeService
+```
+
+Make *PrimeService* the current directory and run [`dotnet new classlib -lang VB`](../tools/dotnet-new.md) to create the source project. Rename *Class1.VB* to *PrimeService.VB*. To use test-driven development (TDD), you create a failing implementation of the `PrimeService` class:
+
+```vb
+Imports System
+
+Namespace Prime.Services
+    Public Class PrimeService
+        Public Function IsPrime(candidate As Integer) As Boolean
+            Throw New NotImplementedException("Please create a test first")
+        End Function
+    End Class
+End Namespace
+```
+
+Change the directory back to the *unit-testing-vb-using-stest* directory. Run [`dotnet sln add .\PrimeService\PrimeService.vbproj`](../tools/dotnet-sln.md)
+to add the class library project to the solution.
+
+ ### Install the NUnit project template
+
+ The NUnit test project templates need to be installed before creating a test project. This only needs to be done once. Run [`dotnet new -i NUnit3.DotNetNew.Template`](../tools/dotnet-new.md) to install the NUnit templates.
```

Comment:
Same comment as the C# project: Let's clarify that this is a per machine install.